src/Storages/ObjectStorage/Azure/Configuration.cpp

```diff
@@ -444,6 +459,22 @@ void StorageAzureConfiguration::addStructureAndFormatToArgsIfNeeded(
     }
 }
 
+void StorageAzureConfiguration::getTableFunctionArguments(ASTs & args) const
+{
+    if (!args.empty())
+    { /// Just check
```

Comment:
I would just remove this comment

Reply:
Actually, if you implement this function in the way I suggested in https://github.com/Altinity/ClickHouse/pull/615#discussion_r1949607293, perhaps this whole if statement is no longer necessary?